lib/components/StatusTag/StatusTag.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from "react";
+
+import { StatusTagProps } from "./StatusTag.types";
+import {
+  IconWrapper,
+  StatusTagWrapper,
+  StyledOverlineLarge,
+} from "./StatusTag.styled";
+
+const StatusTag: FC<StatusTagProps> = ({
+  label,
+  size = "medium",
+  variant = "neutral",
+  icon,
+}) => {
+  return (
+    <StatusTagWrapper variant={variant} size={size}>
+      {icon && (
+        <IconWrapper variant={variant} label={label}>
+          {icon}
+        </IconWrapper>
+      )}
+      <StyledOverlineLarge variant={variant} size={size}>
+        {label}
```

Comment:
```suggestion
        {children}
```
let's make better use of React's APIs and use children here, it also makes more sense semantically. that way we can render more than text here as well

Reply:
I can change, no problem. But, as it happened with the icon, I won't be able to control the text size anymore, so I think we should ditch the "size" param altogether, what do you think?